components/upload/index.en-US.md

```diff
@@ -34,6 +34,7 @@ Uploading is the process of publishing information (web pages, text, pictures, v
 | itemRender | Custom item of uploadList | (originNode: ReactElement, file: UploadFile, fileList?: object\[]) => React.ReactNode | - | 4.7.0 |
 | listType | Built-in stylesheets, support for three types: `text`, `picture` or `picture-card` | string | `text` |  |
 | maxCount | Limit the number of uploaded files. Will replace current one when `maxCount` is `1` | number | - | 4.10.0 |
+| propagateDropEvent | Controls whether to propagate the event on onFileDrop function, this can be useful i.e if you have usecases where you have nested Uploads and you want to prevent double triggering of events. | boolean | true |  |
```

Comment:
It controls whether to propagate the following events:
```
onDrop={onFileDrop}
onDragOver={onFileDrop}
onDragLeave={onFileDrop}
```
Might make sense to modify the description to say something like "Controls whether to propagate the drag/drop events, ..." to clarify things a bit